### PR TITLE
Feature/ce 267 crear nota

### PR DESCRIPTION
**Pull request by Tony-L-93**
Se crea todos los packages faltantes. Se configura propierties para que se conecte a la base de datos. Se crea puede crear grade ya que se implentó controller, service, repository.

### CODE REVIEW

File: src/main/java/click/escuela/grade/rest/GradeController.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	@Operation(summary = "Get all the grades", responses = {
+			@ApiResponse(responseCode = "200", content = @Content(mediaType = "application/json", schema = @Schema(implementation = GradeDTO.class))) })
+	@GetMapping(value = "/getAll", produces = { MediaType.APPLICATION_JSON_VALUE })
+	public ResponseEntity<List<GradeDTO>> getStudents() {
+		return ResponseEntity.status(HttpStatus.ACCEPTED).body(gradeService.findAll());
+	}
+
+	@Operation(summary = "Create grade", responses = {
+			@ApiResponse(responseCode = "200", content = @Content(mediaType = "application/json")) })
+	@PostMapping(value = "/{studentId}", produces = { MediaType.APPLICATION_JSON_VALUE })
```
Comment (clickescuela) [inline src/main/java/click/escuela/grade/rest/GradeController.java]: studentId should not be in the path, I would add it inside the gradeApi

File: src/main/java/click/escuela/grade/model/Grade.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	@Column(name = "id", columnDefinition = "BINARY(16)")
+	@GeneratedValue(generator = "uuid2")
+	@GenericGenerator(name = "uuid2", strategy = "uuid2")
+	private UUID id;
+
+	@Column(name = "name", nullable = false)
+	private String name;
+
+	@Column(name = "course", nullable = false)
+	private String course;
```
Comment (clickescuela) [inline src/main/java/click/escuela/grade/model/Grade.java]: Sorry I made a mistake, can you change "course" to "courseId"?

File: src/main/java/click/escuela/grade/model/Grade.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	
+	@Column(name = "subject", nullable = false)
+	private String subject;
+	
+	@Column(name = "type", nullable = false)
+	private String type;
```
Comment (clickescuela) [inline src/main/java/click/escuela/grade/model/Grade.java]: In the field type Can we change String to Enum? If this is very difficult we can add it in another story

**Review by clickescuela — CHANGES_REQUESTED** [review]

File: src/main/java/click/escuela/grade/model/Grade.java (hunk 3 above)
Comment (Tony-L-93) [reply src/main/java/click/escuela/grade/model/Grade.java]: But what kind of "type" do youn want? Give me some examples so I can put in TypeEnum.

File: src/main/java/click/escuela/grade/enumerator/TypeEnum.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package click.escuela.grade.enumerator;
+
+public enum TypeEnum {
```
Comment (clickescuela) [inline src/main/java/click/escuela/grade/enumerator/TypeEnum.java]: TypeEnum is very generic Can you put GradeType?

File: src/main/java/click/escuela/grade/mapper/Mapper.java [hunk 5]
```diff
@@ -20,7 +21,11 @@ private Mapper() {
 	private static ModelMapper modelMapper = new ModelMapper();
 
 	public static Grade mapperToGrade(GradeApi gradeApi) {
-		return modelMapper.map(gradeApi, Grade.class);
+		Grade grade=modelMapper.map(gradeApi, Grade.class);
```
Comment (clickescuela) [inline src/main/java/click/escuela/grade/mapper/Mapper.java]: apply indentation please

**Review by clickescuela — CHANGES_REQUESTED** [review]

**Review by clickescuela — APPROVED** [review]

**Pull request merged**